Sources/SnapshotPreviewsCore/UIKitRenderingStrategy.swift

```diff
@@ -30,41 +30,52 @@ public class UIKitRenderingStrategy: RenderingStrategy {
   }
 
   private let window: UIWindow
+  private var geometryUpdateError = false
```

Comment:
When is this reset if the same render instance is used to render multiple views?